project/src/main/webapp/script.js

```diff
@@ -40,13 +39,24 @@ function fetchFromQuery() {
   const placesDiv = document.getElementById('place');
   displayResultsPage();
   const map = createMap();
-  fetch('/query?' + params).then(response => response.json()).then((places) => {
-    places.forEach((singlePlace) => {
-      placesDiv.appendChild(createPlaceElement(singlePlace));
-      addPlaceMarker(map, singlePlace)
-    });
-    displayAfterResults();
-  });
+  fetch('/query?' + params)
+      .then(response => response.json())
+      .then((places) => {
+        places.forEach((singlePlace) => {
+          placesDiv.appendChild(createPlaceElement(singlePlace));
+          addPlaceMarker(map, singlePlace)
+        });
+        displayAfterResults();
+        if (places.length < 3) {
+          document.getElementById('message-container').innerHTML =
+              'Your search had ' + places.length + ' results. You are welcome to try again, \
```

Comment:
"only" X results ?

Reply:
Hahaha I also thought about it, but because it also applies to 0 I thought it wouldn't fit. 
Do you think we should divide to different cases? I can do that

Reply:
I think that two different strings is the way to go.

Reply:
done